### DISCUSSION TITLE
Image offset / overlap other components

### DISCUSSION THREAD

**Discussion by TaseerAhmad** [mjmlio/mjml#2867] [Q&A]
Hello!

Firstly, thank you for creating a superb solution for email designs, and I have been using solely MJML to create promotional emails for my company. However, as the design becomes more complex, I am finding certain designs near impossible to implement. One such obstacle I am facing is to 'offset' or make an image overlap mj-columns, as seen in the picture below:
![1](https://github.com/mjmlio/mjml/assets/46527141/de5b2b28-8cfa-4ca7-98ae-12d72ed67416)

I am no UI / front-end developer so I may be lacking knowledge of CSS tricks to get this done for email. Just requesting a small help in a form of a general advice I can follow to achieve my objective. Thank you!

**Comment by iRyusa** [comment]
You could use something like https://www.goodemailcode.com/email-enhancements/faux-absolute-position.html technique.
